src/components/home/HomePage.js

```diff
@@ -89,6 +105,24 @@ const HomePage = () => {
             )
           )}
         </StyledHomePosts>
+
+        <StyledHomeSinglePostPostOption postOptionIsOpen={postOptionIsOpen}>
+          <span aria-hidden="true" onClick={() => setPostOptionIsOpen(false)}>
+            Report ...
+          </span>
+          <span aria-hidden="true" onClick={() => setPostOptionIsOpen(false)}>
+            Hide
+          </span>
+          <span aria-hidden="true" onClick={() => setPostOptionIsOpen(false)}>
```

Comment:
It is better to use `button` instead of `span` when you have a click event. so there will be no need for that `aria-hidden="true"` prop to bypass the eslint a11y error.
use a button and do your styles on the button. you can create a `GhostButton` component that has no background and border and use it where ever you want, and apply your styles on it.